Demo/src/main/java/com/paypal/android/api/services/SDKSampleServerAPI.kt

```diff
@@ -56,10 +61,18 @@ class SDKSampleServerAPI(baseUrl: String) {
         suspend fun authorizeOrder(@Path("orderId") orderId: String): Order
     }
 
-    private val service: RetrofitService
+    private val serviceMap: Map<Merchant, RetrofitService>
 
     init {
+        val serviceMap = mutableMapOf<Merchant, RetrofitService>()
```

Comment:
What does `serviceMap` mean? Can this be something more descriptive

Reply:
The equivalent in Swift would be `serviceDictionary`. The lookup key is the `Merchant` enum, and the value is a Retrofit instance that makes HTTP requests using the merchant's associated `baseUrl`.

The sample merchant server has a base route for each supported integration pattern e.g. Direct, Connected Path, Managed path. This way we can support multiple integration methods using a single server.

Reply:
Service is also a concept taken from the [Retrofit docs](https://square.github.io/retrofit/). We could call `RetrofitService` something else, but `SampleMerchantServerService` doesn't feel right even though it's technically what it is. Here's the whole Retrofit integration in [SDKSamplerServerAPI.kt](https://github.com/paypal/Android-SDK/blob/c2ebc91f0abc68174b6236a1ce9b6a17ad9737e7/Demo/src/main/java/com/paypal/android/api/services/SDKSampleServerAPI.kt#L37).